canonical-web-and-design/ubuntu.com
# Removes cloud pricing survey banner

## Done

- Removes cloud pricing survey banner

## QA

Go to the following pages and see the cloud pricing banner has been removed:
https://ubuntu-com-11674.demos.haus/ceph
https://ubuntu-com-11674.demos.haus/ceph/what-is-ceph
https://ubuntu-com-11674.demos.haus/cloud
https://ubuntu-com-11674.demos.haus/cloud/hybrid-cloud
https://ubuntu-com-11674.demos.haus/cloud/multi-cloud
https://ubuntu-com-11674.demos.haus/cloud/private-cloud
https://ubuntu-com-11674.demos.haus/cloud/public-cloud
https://ubuntu-com-11674.demos.haus/kubernetes
https://ubuntu-com-11674.demos.haus/kubernetes/what-is-kubernetes
https://ubuntu-com-11674.demos.haus/openstack
https://ubuntu-com-11674.demos.haus/openstack/what-is-openstack

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5453
